tools/config.py

```diff
@@ -0,0 +1,39 @@
+import os
+import boto3
+import configparser
+
+PAGE_CROPPPING_BUCKET = "image-processing.bdrc.io"
+BDRC_ARCHIVE_BUCKET = "archive.tbrc.org"
+LAYOUT_ANALYSIS_BUCKET = "image-processing.openpecha"
+
+
+aws_credentials_file = os.path.expanduser("~/.aws/credentials")
+config = configparser.ConfigParser()
+config.read(aws_credentials_file)
+
+
+session1 = boto3.Session(
```

Comment:
rather than session1 i would make it more self explanatory like page_detection_session. same goes to s3_client. numbering can be interpreted in any way